test/components/TaskListTest.js

```diff
@@ -43,68 +43,33 @@ describe('TaskList', () => {
 
   it('does not show task that is ignored', () => {
     assert.equal(1, renderedDOM().querySelectorAll('#pull-163031382').length)
-    store.dispatch({
-      type: 'TASKS_SELECT', task: { storageKey: 'pull-163031382' },
-    })
-    store.dispatch({ type: 'TASKS_IGNORE' })
+    store.dispatch({ type: 'TASKS_IGNORE', task: { storageKey: 'pull-163031382' } })
     assert.equal(0, renderedDOM().querySelectorAll('#pull-163031382').length)
-
-    // Reset state
-    store.dispatch({
-      type: 'TASKS_SELECT', task: { storageKey: 'pull-163031382' },
-    })
-    store.dispatch({ type: 'TASKS_RESTORE' })
+    store.dispatch({ type: 'TASKS_RESTORE', task: { storageKey: 'pull-163031382' } })
```

Comment:
I like that we're using the action part now so we don't have to rely on having previously sent a different action to affect a task. It's nice to have it all in one command.